pkg/handlers/update.go

```diff
@@ -60,13 +60,15 @@ func (h *Handler) AppUpdateCheck(w http.ResponseWriter, r *http.Request) {
 		}
 
 		// preflights reporting
-		go func() {
-			isUpdate := true
-			err = reporting.SendPreflightInfo(foundApp.ID, int(foundApp.CurrentSequence), skipPreflights, isUpdate)
-			if err != nil {
-				logger.Debugf("failed to update preflights reports: %v", err)
-			}
-		}()
+		if deploy {
+			go func() {
+				isCli := true
+				err = reporting.SendPreflightInfo(foundApp.ID, int(foundApp.CurrentSequence), skipPreflights, isCli)
```

Comment:
`foundApp.CurrentSequence` is not the latest downloaded sequence here because that happens inside a goroutine in the background and most probably after this executed.